ui/desktop/app/routes/scopes/scope/projects/targets/target.js

```diff
@@ -86,6 +85,26 @@ export default class ScopesScopeProjectsTargetsTargetRoute extends Route {
     return true;
   }
 
+  /**
+   * Removes any errors from target model when transitioning away
+   * from this route and sets isConnectionError to false if true.
+   * @returns {boolean}
+   */
+  @action
+  willTransition() {
+    /* eslint-disable-next-line ember/no-controller-access-in-routes */
```

Comment:
As mentioned below, we shouldn't avoid this eslint rule, perhaps looks for an alternative way. What we are trying to achieve here? Happy to help to look for an alternative.